ONSdigital/eq-compose
# Add Firebase configuration for author

### What is the context of this PR?
Adds ability to provide Firebase authentication for Author as implemented in https://github.com/ONSdigital/eq-author/pull/273

### How to review
- Run and verify auth is disabled by default
- Get some [firebase](https://firebase.google.com) credentials and add them to a `.env` file in the same directory as eq.yml:
  ```
  AUTHOR_ENABLE_AUTH=true
  AUTHOR_FIREBASE_PROJECT_ID=<your project id>
  AUTHOR_FIREBASE_API_KEY=<your api key>
  AUTHOR_FIREBASE_MESSAGING_SENDER_ID=<your sender id>
  ```
- Run and verify signing in via Firebase works
